client/blocks/importer/wordpress/upgrade-plan/use-default-hosting-details.tsx

```diff
@@ -0,0 +1,53 @@
+import { next, trendingUp, chartBar } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import cwvtechReportJson from './cwvtech-report.json';
+import type { ReactNode } from 'react';
+
+type DefaultHostingDetails = {
+	[ key: string ]: {
+		title: string;
+		description: string | ReactNode;
+		icon: JSX.Element;
+	};
+};
+
+const higherSpeedPercentage = Math.round(
+	( cwvtechReportJson[ 'WordPress.com' ].goodLCP - cwvtechReportJson[ 'WordPress' ].goodLCP ) * 100
+);
+
+const fasterResponsePercentage = Math.round( cwvtechReportJson[ 'WordPress.com' ].goodFID * 100 );
+
+const higherAvailabilityPercentage = 3;
+
+export const useDefaultHostingDetails = (): DefaultHostingDetails => {
```

Comment:
I moved it from the constants file because this has some calcs and doesn't seem to fit well in the constants. Also, in the future, all the data from the JSON will probably be dynamically fetched.

I also took the opportunity to change back to the `useTranslate` hook, since [it seems to fix some issues](https://github.com/Automattic/wp-calypso/pull/89857).